index.html

```diff
@@ -1,14 +1,113 @@
 <!DOCTYPE html>
 <html>
 <head>
-<link rel="stylesheet" type="text/css" href="style.css">
+ <link rel="stylesheet" type="text/css" href="style.css">
+ <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+  <title>medhub</title>
 </head>
 <body>
-
+<div class="container">
 	<img src="https://www.medhub.com/wp-content/uploads/2016/10/Medhub_Logo_CMYK.png" width="200px" height="auto">
-	<h1 class="header">My medHub</h1>
+	<h1 class="header">VIAGRA</h1>
+</div>
+	<div class="info">
+		<div> 
+			<p>Despre Viagra</p>
+		</div>
+		<div class="vl"></div>
+		<div>
+			<p>Contacteaza-ne</p>
+		</div>
+		<div class="vl"></div>
+		<div class="romania">
```

Comment:
Class names should be a brief short description of something that contains the element, so adding romania as a class name, it's a bit confusing. Try to think about what romania represents for this project, if it's a title, than maybe a class name of title would be more handy. Try to use an ul or check this link for a proper navbar : https://getbootstrap.com/docs/4.0/components/navbar/